documentation/modules/mirrormaker/con-mirrormaker-replication.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// assembly-mirrormaker.adoc
+
+[id='con-mirrormaker-{context}']
+= {MirrorMaker} data replication
+
+{MirrorMaker} consumes messages from a source Kafka cluster and writes them to a target Kafka cluster.
+
+{MirrorMaker} uses:
+
+* Source cluster configuration to consume data from the source cluster
+* Target cluster configuration to output data to the target cluster
+
+{MirrorMaker} is based on the Kafka Connect framework, _connectors_ managing the transfer of data between clusters.
+{MirrorMaker} `SourceConnector` replicates topics from a source cluster to a target cluster.
+
+The process of _mirroring_ data from one cluster to another cluster is asynchronous.
+The recommended pattern is for messages to be produced locally alongside the source Kafka cluster, then consumed remotely close to the target Kafka cluster.
+
+{MirrorMaker} can support more than two active clusters.
+
+.Replication across two clusters
+image::mirrormaker.png[{MirrorMaker} replication]
+
+= Cluster configuration
+
+You can use {MirrorMaker} in _active/passive_ or _active/active_ cluster configurations.
+
+* In an _active/passive_ configuration, the data from an active cluster is replicated in a passive cluster, which remains on standby, for example, for data recovery in the event of system failure.
+* In an _active/active_ configuration, both clusters are active and provide the same data simultaneously, which is useful if you want  to make the same data available locally in different geographical locations.
+
+The expectation is that producers and consumers connect to active clusters only.
+
+== Bidirectional replication
+
+The {MirrorMaker} architecture supports bidirectional replication in an _active/active_ cluster configuration.
+
+Each cluster replicates the data of the other cluster using the concept of _source_ and _remote_ topics.
+As the same topics are stored in each cluster, remote topics are automatically renamed by {MirrorMaker} to represent the source cluster.
+
+.Topic renaming in a bidirectional architecture
+image::mirrormaker-renaming.png[{MirrorMaker} bidirectional architecture]
+
+By flagging the originating cluster, topics are not replicated back to that cluster.
+
+The concept of replication through _remote_ topics is useful when configuring an architecture that requires data aggregation.
+Consumers can subscribe to source and remote topics within the same cluster, without the need for a separate aggregation cluster.
+
+== Topic configuration synchronization
+
+Topic configuration is automatically synchronized between source and target clusters.
+By synchronizing configuration properties, the need for rebalancing is reduced.
+
+== ACL access to remote topics
+
+If `SimpleAclAuthorizer` is being used, ACL rules that manage access to brokers also apply to remote topics.
```

Comment:
This does nto work with User Operator. So I think it should be mentioned as not supported here? Or at least just remove this whole subchapter.

Reply:
Maybe someone might wonder what happens with ACL access? Perhaps it's better to say that it is not supported if User Operator is being used?

Reply:
> == ACL access to remote topics
> 
> ACL access to remote topics is possible if you are not using the User Operator.
> 
> If `SimpleAclAuthorizer` is being used, ACL rules that manage access to brokers also apply to remote topics.
> Users that can read a source topic can read its remote equivalent.
> 
> NOTE: {oauth} authorization does not support access to remote topics in this way.

Reply:
Ok, makes sense. I think we should call it more something like _ACL rules synchronization_. People will not understand what exactly it means under _ACL access to remote topics_.